src/Infrastructure/TimeMgr/src/ESMCI_Alarm.C

```diff
@@ -129,59 +128,61 @@ int Alarm::count=0;
       sprintf(alarm->name, "Alarm%3.3d", alarm->id);
     }
 
-    if (ringTime != ESMC_NULL_POINTER) {
-      alarm->ringTime = alarm->prevRingTime = alarm->firstRingTime = *ringTime;
```

Comment:
- [ ] I think it's a problem that `firstRingTime` is no longer set: it's still referenced in `ringerOff` (see https://github.com/esmf-org/esmf/pull/340#issuecomment-2566736806)